README.md

```diff
@@ -198,6 +198,8 @@ The StellarGraph library currently includes the following algorithms for graph m
 | Relational Graph Convolutional Network [11] | The RGCN algorithm performs semi-supervised learning for node representation and node classification on knowledge graphs. RGCN extends GCN to directed graphs with multiple edge types and works with both sparse and dense adjacency matrices.|
 | ComplEx[12] | The ComplEx algorithm computes embeddings for nodes (entities) and edge types (relations) in knowledge graphs, and can use these for link prediction |
 | GraphWave [13] | GraphWave calculates unsupervised structural embeddings via wavelet diffusion through the graph. |
+| Deep Graph Infomax [14] | Deep Graph Infomax trains GNNs unsupervised to maximize the information in the final node features. |
```

Comment:
The "...to maximize the information in the final node features" part doesn't make much sense by itself. You might have to re-write.

Reply:
how about:

"Deep Graph Infomax trains unsupervised GNNs to maximize the shared information between node level and graph level features."

Reply:
I think that's better although it still requires the reader to have some understanding of what it all means. For now, I think your last suggestion is better than the one before.